lib/msal-common/src/request/ScopeSet.ts

```diff
@@ -125,35 +169,47 @@ export class ScopeSet {
 
     /**
      * Removes element from set of scopes.
-     * @param scope 
+     * @param scope
      */
     removeScope(scope: string): void {
         if (StringUtils.isEmpty(scope)) {
-            throw ClientAuthError.createRemoveEmptyScopeFromSetError(scope);
+            throw ClientAuthError.createRemoveEmptyScopeFromSetError(
+                scope
+            );
         }
         this.scopes.delete(scope.trim().toLowerCase());
     }
 
     /**
      * Combines an array of scopes with the current set of scopes.
-     * @param otherScopes 
+     * @param otherScopes
      */
     unionScopeSets(otherScopes: ScopeSet): Set<string> {
         if (!otherScopes) {
-            throw ClientAuthError.createEmptyInputScopeSetError(otherScopes);
+            throw ClientAuthError.createEmptyInputScopeSetError(
+                otherScopes
+            );
         }
-        return new Set<string>([...otherScopes.asArray(), ...Array.from(this.scopes)]);
+        return new Set<string>([
+            ...otherScopes.asArray(),
+            ...Array.from(this.scopes)
+        ]);
     }
 
     /**
      * Check if scopes intersect between this set and another.
-     * @param otherScopes 
+     * @param otherScopes
      */
     intersectingScopeSets(otherScopes: ScopeSet): boolean {
         if (!otherScopes) {
-            throw ClientAuthError.createEmptyInputScopeSetError(otherScopes);
+            throw ClientAuthError.createEmptyInputScopeSetError(
+                otherScopes
+            );
         }
-        return this.unionScopeSets(otherScopes).size < (this.scopes.size + otherScopes.getScopeCount());
+        return (
+            this.unionScopeSets(otherScopes).size <
+                   this.scopes.size + otherScopes.getScopeCount()
+        );
```

Comment:
Revert this change, we don't want OOO messing up here. I know '+' is higher order than '<' but this change doesn't help us ensure that.

Reply:
auto correct by "prettify". We can change the linter rules as we see it before final merge.

Reply:
Changed and committed